diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8ee07c2..7259714 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -98,6 +98,18 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "aarc
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
   set(FOXGLOVE_SDK_PLATFORM "x86_64-unknown-linux-gnu")
   set(FOXGLOVE_SDK_SHA "0b9d348df3a8e98d2c2cee2360cc9f52e83bb3445044debd771d1b46dc358be4")
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "arm64")
+  set(FOXGLOVE_SDK_PLATFORM "aarch64-apple-darwin")
+  set(FOXGLOVE_SDK_SHA "c9dfa5061d72795af2620c2840a85e695b9dacce19475eaec8caf0102c4940ef")
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+  set(FOXGLOVE_SDK_PLATFORM "x86_64-apple-darwin")
+  set(FOXGLOVE_SDK_SHA "3531502efd958be3f9dca06a099c167512dd3006b98ac3eecf8250d376eaa16b")
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
+  set(FOXGLOVE_SDK_PLATFORM "aarch64-pc-windows-msvc")
+  set(FOXGLOVE_SDK_SHA "da293bf4dac2b973b7c72d1d368932f7f4f989fcac9311d529b93c7d2b84eae9")
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows" AND CMAKE_SYSTEM_PROCESSOR STREQUAL "AMD64")
+  set(FOXGLOVE_SDK_PLATFORM "x86_64-pc-windows-msvc")
+  set(FOXGLOVE_SDK_SHA "09d2ff5dbb6a5216a303f9470fbec186c0ebd849713570d6b8145a9d22d804af")
 else()
   message(FATAL_ERROR "Unsupported platform/architecture combination: ${CMAKE_SYSTEM_PROCESSOR}-${CMAKE_SYSTEM_NAME}")
 endif()
diff --git a/src/message_definition_cache.cpp b/src/message_definition_cache.cpp
index ef72c9c..ed85745 100644
--- a/src/message_definition_cache.cpp
+++ b/src/message_definition_cache.cpp
@@ -3,6 +3,7 @@
 #include <filesystem>
 #include <fstream>
 #include <functional>
+#include <map>
 #include <optional>
 #include <regex>
 #include <set>
